pkgs/games/angband/default.nix

```
@@ -11,8 +15,15 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-Fp3BGCZYYdQCKXOLYsT4zzlibNRlbELZi26ofrbGGPQ=";
   };
 
+  configureFlags = lib.optional enableSdl2 "--enable-sdl2";
   nativeBuildInputs = [ autoreconfHook ];
-  buildInputs = [ ncurses5 ];
+  buildInputs = [ ncurses5 ] ++ lib.optionals enableSdl2 [
+    SDL2
+    SDL2_image
+    SDL2_sound
+    SDL2_mixer
+    SDL2_ttf
+  ];
   installFlags = [ "bindir=$(out)/bin" ];
```

```suggestion
  buildInputs = [ ncurses5 ]
  ++ lib.optionals enableSdl2 [
    SDL2
    SDL2_image
    SDL2_sound
    SDL2_mixer
    SDL2_ttf
  ];

  configureFlags = lib.optional enableSdl2 "--enable-sdl2";

  installFlags = [ "bindir=$(out)/bin" ];
```

lets use this opportunity to add some new line here.

---

The preferred way is then to group "by phase"?